### DISCUSSION TITLE
bizonylat verzió lekérdezés

### DISCUSSION THREAD

**Discussion by rrd108** [nav-gov-hu/M2M#56] [Kérdések]
van tervben bizonylat verzió lekérdezés?

Ha kijön egy új verzió az ÁNYK-s beküldés már nem mindig fogadja el a korábbi verzióval készült bizonylatot.

**Comment by nanassy72** [comment]
Tervezünk egy műveletet a bizonylat verzió lekérdezésére, de ez még átgondolás alatt áll. Amint lesz konkrétum, jelezzük.
